test/choosetests.jl

```diff
@@ -118,7 +118,7 @@ function choosetests(choices = [])
                    "linalg/diagonal", "linalg/pinv", "linalg/givens",
                    "linalg/cholesky", "linalg/lu", "linalg/symmetric",
                    "linalg/generic", "linalg/uniformscaling", "linalg/lq",
-                   "linalg/hessenberg"]
+                   "linalg/hessenberg", "linalg/blas"]
```

Comment:
I've moved the tests to the existing `blas.jl` test file. I'll probably move that file to the linalg tests when this is merged.